Consistently note if constructor can be called w/o new (#19631)
mdn/content: files/en-us/web/javascript/reference/global_objects/aggregateerror/aggregateerror/index.md

---
{{JSRef}}

The **`AggregateError()`** constructor creates an error for several errors that need to be wrapped in a single error.

## Syntax

```js
new AggregateError(errors)
new AggregateError(errors, message)
new AggregateError(errors, message, options)

AggregateError(errors)
AggregateError(errors, message)
AggregateError(errors, message, options)
```

> > **Note:** `AggregateError()` can be called with or without [`new`](/en-US/docs/Web/JavaScript/Reference/Operators/new) — they have the same effect of constructing `AggregateError` instances.

Author: Yes.

> > **Note:** `AggregateError()` can be called with or without [`new`](/en-US/docs/Web/JavaScript/Reference/Operators/new) — they have the same effect of constructing `AggregateError` instances.

Reviewer: So both create a new object?
```suggestion
> **Note:** `AggregateError()` can be called with or without [`new`](/en-US/docs/Web/JavaScript/Reference/Operators/new). Both create a new `AggregateError` instance.
```

> > **Note:** `AggregateError()` can be called with or without [`new`](/en-US/docs/Web/JavaScript/Reference/Operators/new) — they have the same effect of constructing `AggregateError` instances.

Author: Ah yes. That reads better.
